capreolus/collection/robust04.py

```diff
@@ -100,3 +100,53 @@ def download_index(
             print("", file=outf)
 
         return document_dir
+
+
+@Collection.register
+class Robust04Passages(Collection):
+    """
+    TREC Robust04 (TREC disks 4 and 5 without the Congressional Record documents).
+    Splits each document in Robust04 into passages and indexes them separately.
+    """
+
+    module_name = "robust04passages"
+    collection_type = "TrecCollection"
+    generator_type = "DefaultLuceneDocumentGenerator"
+    config_keys_not_in_path = ["path"]
+    config_spec = [ConfigOption("path", "Aquaint-TREC-3-4", "path to corpus")]
+    dependencies = [Dependency(key="task", module="task", name="robust04passages")]
```

Comment:
Using the `robust04passages` collection/benchmark will automatically invoke the task that will chunk the docs into passages.